### DISCUSSION TITLE
Craft 10/Craft All for axes

### DISCUSSION THREAD

**Discussion by Xeko-92** [sunflower-land/sunflower-land#2361] [Ideas]
I can't believe I need to make a discussion for this, but how has this not been added yet? Especially with the increase in stock & the toolshed being added? Needing to click that button 300 times per sync is absurd.

**Comment by TheDcol** [comment]
Yes we need it!

It could have been a thing already D:

https://github.com/sunflower-land/sunflower-land/pull/1850#issuecomment-1407787240

**Comment by future2back** [comment]
Come on!
